CHANGELOG_NEXT.md

```diff
@@ -227,6 +227,8 @@ This CHANGELOG describes the merged but unreleased changes. Please see [CHANGELO
 
 * Added `Data.IORef.atomically` for the chez backend.
 
+* Refactor `Uninhabited` implementation for `Data.List.Elem`, `Data.List1.Elem`, `Data.SnocList.Elem` and `Data.Vect.Elem`.
```

Comment:
Would you mind adding a brief sentence explaining why / explaining what the refactor achieves?

Reply:
done